src/main/python/systemml/converters.py

```diff
@@ -212,10 +215,86 @@ def convertToNumPyArr(sc, mb):
     else:
         raise TypeError('sc needs to be of type SparkContext') # TODO: We can generalize this by creating py4j gateway ourselves
 
+
+#   Load an image converting from grayscale or alpha as needed.
+def load_image(filename, color=True):
+    """
+    Input Parameters
+    ----------
+    filename : string
+    color : boolean
+        flag for color format. True (default) loads as RGB while False
+        loads as intensity (if image is already grayscale).
+
+    Returns
+    -------
+    image : an image with type np.float32 in range [0, 1]
+        of size (H x W x 3) in RGB or
+        of size (H x W x 1) in grayscale.
+    """
+    import cv2
+    image = skimage.img_as_float(cv2.imread(filename))
```

Comment:
I understand the intent here is to be consistent with BVLC Caffe, but skimage is an additional dependency. Do you think it is necessary ?